cmd/gpu_plugin/README.md

```diff
@@ -56,6 +56,8 @@ For workloads on different KMDs, see [KMD and UMD](#kmd-and-umd).
 |:---- |:-------- |:------- |:------- |
 | -enable-monitoring | - | disabled | Enable '*_monitoring' resource that provides access to all Intel GPU devices on the node, [see use](./monitoring.md) |
 | -resource-manager | - | disabled | Enable fractional resource management, [see use](./fractional.md) |
+| -health-management | - | disabled | Enable health management by requesting data from oneAPI/Level-Zero interface. Requires [GPU Level-Zero](../gpu_levelzero/) sidecar. |
+| -wsl | - | disabled | Adapt plugin to run in the WSL environment. Requires [GPU Level-Zero](../gpu_levelzero/) sidecar. |
```

Comment:
Potential alternative descriptions:
```suggestion
| -health-management | - | disabled | Disable GPU devices when metrics indicate them not being healthy.  Requires [GPU Level-Zero](../gpu_levelzero/) sidecar. |
| -wsl | - | disabled | Use / scan for WSL environment pass-through GPU devices. Requires [GPU Level-Zero](../gpu_levelzero/) sidecar. |
```

(If descriptions changed here, would need to be changed also elsewhere.)

Reply:
I kept them as they are. But I did add a "health management" section with some explanations.